common/walker/src/report.rs

```diff
@@ -124,8 +128,83 @@ impl SplitScannerError for Result<Report, ScannerError> {
 }
 
 /// Handle the report
-pub async fn handle_report(report: Report) -> anyhow::Result<()> {
-    // FIXME: this is a very simplistic version of handling the error
-    log::info!("Import report: {report:#?}");
+pub async fn handle_report(report: Report, report_path: Option<String>, report_type: String) -> anyhow::Result<()> {
+    if report.messages.is_empty() {
+        log::info!("This report contains no error messages and does not require the generation of an error report");
+        return Ok(());
+    }
+    let template_file_path = env::var_os("TEMPLATE_FILE");
```

Comment:
Is there a reason this isn't handled as a clap argument? Including adding the `env`  attribute.

Reply:
I think the HTML template feature do not needs expose to user.

Reply:
This is still using `env::var_os` somewhere inside the code. Why can't this be handled in the place we have the other options, using `clap`, using the `long` and `env` option?